### PR TITLE
Add file with scenario endpoints and add a file link in Read.me

### PR DESCRIPTION
**Pull request by Annibezh**
Add file with scenario endpoints to repository and add a file link in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,12 @@ There are four modules:
   to project dependencies. It simplifies creation of REST API and provides built-in JWT 
   authentication.
 
+### Main scenario endpoints
+
+You can find main scenario endpoints in [google doc](https://docs.google.com/document/d/1WVX6a76JkLmzUeJz2cZbO1WvXqWsY_XO2sA3a15qgIQ/edit#heading=h.yqb5kvpzfmry)
+
+We also recomend to see some [curl examples](https://github.com/future-stardust/url-shrtnr-le-gushqua/blob/main/manualTests/http.http)
```
Comment (quppeq) [inline README.md:43]: maybe better new md file in repository?

**Comment by quppeq** [comment]
please remove blank pdf file

File: README.md (hunk 1 above)
Comment (quppeq) [reply README.md:43]: or wiki

**Review by quppeq — CHANGES_REQUESTED** [review]
Please create 'handbook' in repository

File: README.md (hunk 1 above)
Comment (Annibezh) [reply README.md:44]: Have created Scenario endpoints.md

File: Scenario endpoints.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    Content-Type: application/json
+
+    {"username":"test","password":"test"}
+
+
+
+### Create an URL alias (with random generated alias name)
+
+    POST localhost:8080/urls/shorten
+    #/shorten
```
Comment (quppeq) [inline Scenario endpoints.md:32]:
```suggestion
    curl localhost:8080/urls/shorten 
      -X POST 
      -H 'Content-Type:application/json' 
      -H 'Authorization: Bearer <TOKEN FROM THE LOGIN RESPONSE>’ 
      -d '{"url":"https://www.google.com"}'
```

File: Scenario endpoints.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    Content-Type: application/json
+    Authorization: Bearer <TOKEN FROM THE LOGIN RESPONSE>
+
+
+
+### Redirect by a shortened URL
+
+    GET localhost:8080/r/gist
+    #/shorten
+    Content-Type: application/json
```
Comment (quppeq) [inline Scenario endpoints.md:65]:
```suggestion
```

File: Scenario endpoints.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    Authorization: Bearer <TOKEN FROM THE LOGIN RESPONSE>
```
Comment (quppeq) [inline Scenario endpoints.md:66]:
```suggestion
```

File: Scenario endpoints.md (hunk 3 above)
Comment (quppeq) [inline Scenario endpoints.md:65]:
```suggestion
```

File: Scenario endpoints.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+      -H 'Authorization: Bearer <TOKEN FROM THE LOGIN RESPONSE>’ 
+      -d '{"url":"https://gist.new", "alias":"gist"}'
+
+
+
+### List user’s shortened links
+
+    # curl localhost:8080/urls/sh 
+      -X GET
+      -H ‘Authorization: Bearer <TOKEN FROM THE LOGIN RESPONSE>’
```
Comment (quppeq) [inline Scenario endpoints.md:54]:
```suggestion
      -H 'Content-Type:application/json'
      -H ‘Authorization: Bearer <TOKEN FROM THE LOGIN RESPONSE>’
```

File: Scenario endpoints.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+    #/shorten
+    Content-Type: application/json
+    Authorization: Bearer <TOKEN FROM THE LOGIN RESPONSE>
+
+
+
+### Delete shortened link
+
+    DELETE localhost:8080/urls/gist
+    #/shorten
```
Comment (quppeq) [inline Scenario endpoints.md:73]:
```suggestion
    curl localhost:8080/urls/gist 
      -X DELETE 
      -H 'Content-Type:application/json' 
      -H 'Authorization: Bearer <TOKEN FROM THE LOGIN RESPONSE>’ 
```

**Review by quppeq — CHANGES_REQUESTED** [review]

File: Scenario endpoints.md (hunk 3 above)
Comment (Annibezh) [reply Scenario endpoints.md]: Thanks! All changes are done

**Review by quppeq — APPROVED** [review]
kinda good, but we can better)

Next "typo pr" fix it. 
Now i approve this)

**Review by quppeq — APPROVED** [review]
Good!
Approve

**Review by quppeq — APPROVED** [review]
its better and better!

maybe if i be good in markdown, i changes this, 
but i bad in markdown =)

Approve

**Comment by quppeq** [comment]
CI doesn`t start -_-

**Review by quppeq — APPROVED** [review]
uuuuuu

**Review by quppeq — APPROVED** [review]

**Pull request merged**